### PR TITLE
Bump tinc to version 1.1 commit 3ee0d5dd

### PR DESCRIPTION
**Pull request by ErwanMAS**
Maintainer: @zioproto 
Compile tested: openwrt 19.07  cross compile to ath79
Run tested: openwrt 19.07 ath79 WR 1043 ND

Description:
     bump version to 1.1
   
ps: @zioproto  i can become maintainer if you agree

### CODE REVIEW

File: net/tinc/files/tinc.init [hunk 1]
```diff
@@ -171,7 +217,7 @@ start_instance() {
 	append_params "$s" logfile debug
 
 	SERVICE_PID_FILE="/var/run/tinc.$s.pid"
-	service_start $BIN -c "$TMP_TINC/$s" -n $s $ARGS --pidfile="$SERVICE_PID_FILE"
+	service_start $BIN -c "$TMP_TINC/$s" $ARGS --pidfile="$SERVICE_PID_FILE"
```
Comment (zioproto) [inline net/tinc/files/tinc.init:220]: Why you are dropping `-n $s`. You plan on dropping support for different network configurations ?
Are you aware of:
https://github.com/openwrt/packages/issues/9914#issuecomment-702972496 ?

File: net/tinc/Makefile [hunk 2]
```diff
@@ -8,24 +8,30 @@
...
 include $(INCLUDE_DIR)/package.mk
 
 define Package/tinc
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+liblzo +libopenssl +kmod-tun +zlib
+  DEPENDS:=+kmod-tun +liblzo +libopenssl +librt +zlib
   TITLE:=VPN tunneling daemon
   URL:=http://www.tinc-vpn.org/
   MAINTAINER:=Saverio Proto <zioproto@gmail.com>
```
Comment (zioproto) [inline net/tinc/Makefile:37]: Please update the MAINTAINER with your details.

Comment (zioproto) [reply net/tinc/Makefile:37]: I am not 100% sure how you get merge rights on github for this package. I dont think the line in the Makefile is enough. You would have to follow up on the OpenWrt Developers mailing list to ask what is the actual procedure for this today.
Thanks for stepping in as Maintainer of Tinc

**Review by zioproto — CHANGES_REQUESTED** [review]
Thanks for the patch!  Please look at the changes I requested. thanks

File: net/tinc/files/tinc.init (hunk 1 above)
Comment (ErwanMAS) [reply net/tinc/files/tinc.init:220]: i dont plan to remove for different network configurations .
When you start with `-c`  and `-n`  you have this warning message with tinc 1.1 
```
Both netname and configuration directory given, using the latter...
```

Comment (ErwanMAS) [reply net/tinc/files/tinc.init:220]: by removing -n , i remove only the warning this will not affect multiple network configuration

**Review by ErwanMAS — COMMENTED** [review]
changes of mantainer done

**Review by zioproto — APPROVED** [review]

**Pull request merged**